pkgs/top-level/python-packages.nix

```
@@ -5837,7 +5837,7 @@ in {
 
   phonemizer = callPackage ../development/python-modules/phonemizer { };
 
-  pyopencl = callPackage ../development/python-modules/pyopencl { };
+  pyopencl = callPackage ../development/python-modules/pyopencl { mesa_drivers = pkgs.mesa.drivers; };
```

sorry for nitpicking but this line is quite long
```suggestion
  pyopencl = callPackage ../development/python-modules/pyopencl {
    mesa_drivers = pkgs.mesa.drivers;
  };
```

---

Done! I amended to the current commit to match the prior comment that @SuperSandro2000 made.